src/common/style/icon_color_css.ts

```diff
@@ -1,5 +1,23 @@
 import { css } from "lit-element";
 
+export interface StateColor {
+  state: string;
+  color?: string;
```

Comment:
Why is color optional ?

Reply:
Defaults to our default active color if not set